src/pages/components/filter-panel.mdx

```diff
@@ -0,0 +1,268 @@
+
+### Single selection
+Use a single-selection filter when the user can pick only one attribute to modify data results. Under the hood, single selection behaves like a radio button.
+
+<Row>
+
+<Column colMd={4} colLg={4}>
+
+![Example image: single selection with no selected filter.](../../images/component/filter-panel/filter-panel-single-selection-example-not-selected.jpg)
+
+<Caption>Example of single selection with no filter item selected.</Caption>
+
+</Column>
+
+<Column colMd={4} colLg={4}>
+
+![Example image: single selection with selected filter.](../../images/component/filter-panel/filter-panel-single-selection-example-selected.jpg)
+
+<Caption>Example of single selection with a filter item selected.</Caption>
+
+</Column>
+
+</Row>
+
+Single-selection supports multiple tiers of content. Second tier content is exposed after the user selects the parent filter, tiered content is indented below the parent filter.
+
+<Row>
+
+<Column colMd={4} colLg={4}>
+
+![Example image: single selection hover state of a parent tier filter item.](../../images/component/filter-panel/filter-panel-single-selection-parent-tier.jpg)
+
+<Caption>Example of single selection hover state of a parent tier filter item.</Caption>
+
+</Column>
+
+<Column colMd={4} colLg={4}>
+
+![Example image: parent tier filter item selected, with child tier items exposed.](../../images/component/filter-panel/filter-panel-single-selection-child-tier.jpg)
+
+<Caption>Example of parent tier filter item selected, with child tier items exposed.</Caption>
+
+</Column>
+
+</Row>
+
+### Multiselect
+Use a multiselect filter when the user has the option to pick more than one attribute to modify the data results. Under the hood, multiselects behave like checkboxes.
+
+<Row>
+
+<Column colMd={4} colLg={4}>
+
+![Example image: multiselect with no selected filter item.](../../images/component/filter-panel/filter-panel-multiselect-not-selected.jpg)
+
+<Caption>Example of multiselect with no selected filter item.</Caption>
+
+</Column>
+
+<Column colMd={4} colLg={4}>
+
+![Example image: multiselect with multiple filter items selected.](../../images/component/filter-panel/filter-panel-multiselect-selected.jpg)
+
+<Caption>Example of multiselect with multiple filter items selected.</Caption>
+
+</Column>
+
+</Row>
+
+### Multiple categories
+A category is a set of filter items within the same topic. For example, “size” is a category and small, medium, large, and extra large are its filter choices. Multiple filter categories may be applied to the same data set. For example, the user can filter by size, color, and price range.
+
+![A visual example of the Filter panel component with multiple categories in context of a page](../../images/component/filter-panel/filter-panel-multiple-categories.jpg)
+
+## Behaviors
+
+### Larger breakpoints (max, xl, lg)
```

Comment:
Is there a custom for which way to show sizes? In the previous section, one of the sentences had sizes ascending upwards and this feels right and more comfortable than going the other way. So maybe matching this here?